syntax = "proto3";

package member;


message MemberRequest {
    string member_id = 1;
}